src/main/python/rlbot/utils/rlbot_exception.py

```diff
@@ -7,10 +7,10 @@ def __init__(self, msg=None):
             msg = "An error occurred attempting to set RLBot configuration on the dll side"
         super(RLBotException, self).__init__(msg)
         self.error_dict = {1: BufferOverfilledError(), 2: MessageLargerThanMaxError(), 3: InvalidNumPlayerError(),
-                           4: InvalidBotSkillError(), 5: InvalidPlayerIndexError(), 6: InvalidName(),
+                           4: InvalidBotSkillError(), 5: InvalidHumanIndex, 6: InvalidName(),
                            7: InvalidTeam, 8: InvalidTeamColor(), 9: InvalidCustomColor, 10: InvalidGameValues,
-                           11: InvalidThrottle, 12: InvalidSteer, 13: InvalidPitch, 14: InvalidRoll,
-                           15: InvalidSteer}
+                           11: InvalidThrottle, 12: InvalidSteer, 13: InvalidPitch, 14: InvalidYaw, 15: InvalidRoll,
+                           16: InvalidPlayerIndexError, 17: InvalidQuickChatPreset, 18: InvalidRenderType}
 
```

Comment:
InvalidPlayerIndexError() is now InvalidPlayerIndexError, (without the parens), is that intended?

Reply:
That's a good point. I refactored a little bit, how do these new changes strike you?

Reply:
Looks great!